### PR TITLE
Add reply_to argument, dropping Django <1.8 in the process

### PR DESCRIPTION
**Pull request by adam-thomas**
@incuna/backend opinions/review please? :)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,16 +8,7 @@ notifications:
...
-    - DJANGO_VERSION='django>=1.6,<1.7'
-    - DJANGO_VERSION='django>=1.5,<1.6'
-    - DJANGO_VERSION='django>=1.4,<1.5'
-matrix:
-  exclude:
-     - env: DJANGO_VERSION='django>=1.4,<1.5'
-       python: 3.4
-     - env: DJANGO_VERSION='django>=1.4,<1.5'
-       python: 3.3
+    - DJANGO_VERSION='django>=1.8,<1.9'
```
Comment (Ian-Foote) [inline .travis.yml]: `django~=1.8.5`

Comment (meshy) [inline .travis.yml]: :+1:

File: incuna_mail.py [hunk 2]
```diff
@@ -9,7 +9,8 @@ def listify(obj):
 
 
 def send(template_name, sender=None, to=None, cc=None, bcc=None, subject='mail',
-         attachments=(), html_template_name=None, context=None, headers=None):
+         attachments=(), reply_to=None, html_template_name=None, context=None,
+         headers=None):
```
Comment (meshy) [inline incuna_mail.py]: If you pop the `reply_to` kwarg to the end, then this will not break for anyone (unwisely) calling this with positional arguments.

Comment (adam-thomas) [inline incuna_mail.py]: oh true!

**Comment by adam-thomas** [comment]
Thanks guys, updated :)

File: incuna_mail.py [hunk 3]
```diff
@@ -36,6 +37,7 @@ def send(template_name, sender=None, to=None, cc=None, bcc=None, subject='mail',
         'bcc': bcc,
         'subject': six.text_type(subject),
         'attachments': attachment_list,
+        'reply_to': reply_to,
```
Comment (meshy) [inline incuna_mail.py]: can we please "listify" this like we have with the `to`, `cc`, etc?

Comment (adam-thomas) [inline incuna_mail.py]: Sure!  (On a side note, wtf is that `map` line?  That's so awkward-looking...)

Comment (meshy) [inline incuna_mail.py]: Take it back! It's a thing of beauty `:P`

**Comment by meshy** [comment]
Ta!

**Comment by adam-thomas** [comment]
Thanks :)

**Pull request merged**